docs/spark/tutorials/index.md

```diff
@@ -1,13 +1,16 @@
 ---
 title:  .NET for Apache Spark tutorials
 description: Explore the .NET for Apache Spark tutorials to learn how to integrate Apache Spark into your .NET applications.
-ms.date: 05/06/2019
+ms.date: 05/17/2019
```

Comment:
I don't think this page can be reached any more with the recent TOC changes. Maybe needs to be removed?